src/components/ebay-button/index.marko

```diff
@@ -12,6 +12,7 @@ export interface ButtonEvent<T extends Event> {
 
 export interface ButtonInput extends Omit<Marko.HTML.Button, `on${string}`> {
```

Comment:
If you change this to extend multiple types then no other overrides are required, and the type will be more precise
```suggestion
export interface ButtonInput extends Omit<Marko.HTML.Button, `on${string}` | "type">, Omit<Marko.HTML.A, `on${string}`> {
```

Reply:
Done